app/proguard.cfg

```diff
@@ -1,4 +1,4 @@
--optimizationpasses 5
+-optimizationpasses 15

```

Comment:
Proguard [stops](http://proguard.sourceforge.net/manual/usage.html#optimizationoptions) when it cannot optimize anymore, so in practice you shouldn't see much difference from a 5-pass run.